docs/bigquery-usage.rst

```diff
@@ -34,31 +34,74 @@ Authorization / Configuration
      >>> client = bigquery.Client()
 
 - Override the credentials inferred from the environment by passing explicit
-  ``credentials`` to one of the alternative classmethods, `from_service_account_json`:
+  ``credentials`` to one of the alternative ``classmethod`` factories,
+  `:meth:gcloud.bigquery.client.Client.from_service_account_json`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_json('/path/to/creds.json')
 
-  or `from_service_account_p12`:
+  or `:meth:gcloud.bigquery.client.Client.from_service_account_p12`:
 
   .. doctest::
 
      >>> from gcloud import bigquery
      >>> client = bigquery.Client.from_service_account_p12('/path/to/creds.p12', 'jrandom@example.com')
 
-- Override the project inferred from the environment by passing an explicit
-  ``project`` to the constructor, or to either of the alternative classmethods:
+
+Projects
```

Comment:
Is there any way we can punt on this concept entirely ? It seems a bit confusing with relatively little benefit.

For example, in Node (just like anything else), we just say "like all the other APIs, tell us which project you want", and keep the key concepts of Dataset, Job, and Table.

@dhermes @stephenplusplus : Thoughts on a top-level "Project" concept in BigQuery?